src/pages/DatasetStatistics.tsx

```diff
@@ -9,15 +9,32 @@ import {find} from 'lodash/fp';
 import {ReadMore} from '../components/ReadMore';
 import {formatDate} from '../libs/utils';
 import {Button} from '@mui/material';
-import {AccessManagement} from '../libs/Models.ts';
+import {History} from 'history';
+import {Dataset, DatasetProperty} from '../types/model';
 
 const LINE = <div style={{borderTop: '1px solid #BABEC1', height: 0}}/>;
 
-export default function DatasetStatistics(props) {
+enum AccessManagement {
```

Comment:
Should this be moved to `Model.ts?` Or this only an UI concept and not part of an API model?

Reply:
It is not a part of the API model I believe. Or at least, it was in a separate file and `model.js` is a generated file. `Model.ts` was conflicting because of the import conflicts in js having issues with casing